dali/pipeline/operators/reader/coco_reader_op.cc

```diff
@@ -428,6 +435,13 @@ void ParseAnnotationFilesHelper(std::vector<std::string> &annotations_filename,
         r.SkipValue();
       }
     }
+    if (skip_empty) {
+      std::vector<std::pair<std::string, int>> non_empty_ids;
```

Comment:
You can `reserve` this to accommodate all non-empty entries.

Reply:
Done